core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -908,6 +912,11 @@ class DAGScheduler(
     val task = event.task
     val stageId = task.stageId
     val taskType = Utils.getFormattedClassName(task)
+    val isSuccess = event.reason == Success
+
+    outputCommitCoordinator.taskCompleted(stageId, event.taskInfo.taskId,
```

Comment:
There should only be two spaces of indentation on the following lines.  Also, it looks like some of the args can be pulled back up without hitting 100 characters.